frontend/__tests__/src/fixtures.ts

```diff
@@ -56,6 +56,7 @@ export const ERROR_MESSAGE_COLOR = 'color: #d32f2f'
 export const ERROR_DATE = '02/03/'
 export const CREATE_NEW_PROJECT = 'Create a new project'
 export const IMPORT_PROJECT_FROM_FILE = 'Import project from file'
+export const EXPORT_EXPIRED_CSV_MESSAGE = 'Export CSV files have been expired'
```

Comment:
update to `The report has been expired, please generate it again.`

Reply:
done